openkfw/TruBudget
# 1953 joi env validation

### Checklist




### Description


### How to test

1. Remove or change env vars for API to something non-valid
2. Start TruBudget with development start script
3. It should produce a correct error and don't start the app

1. Modify env vars in `api/src/envVarsSchema.ts`
2. Run `npm run generate-env-vars-docs` in api
3. Markdown file in ./api/environment-variables.md should update




Closes #1953
